### PR TITLE
feat: 🎸 Circular chart changes

### PR DESCRIPTION
**Pull request by dariuszlacheta**

### CODE REVIEW

File: src/components/ComponentSettings/ComponentSettings.tsx [hunk 1]
```diff
@@ -96,7 +96,7 @@ const ComponentSettings: FC<Props> = ({
     >
       <SettingsContainer isDisabled={!!isDisabled}>
         {settingsComponent
-          ? React.createElement(getSettingsComponent(widgetType), {
+          ? React.createElement(getSettingsComponent('pie'), {
```
Comment (apryka) [inline src/components/ComponentSettings/ComponentSettings.tsx:99]: ?? shouldn't be `widgetType` here?

Comment (maciejrybaniec) [reply src/components/ComponentSettings/ComponentSettings.tsx:99]: +1

File: src/charts/circular/transform.ts [hunk 2]
```diff
@@ -9,13 +9,15 @@ const transform: WidgetTransform<PieChartSettings & DonutChartSettings> = {
...
+      valueMode: settings.valueMode,
     };
   },
-  serializeOut: ({ formatValue }) => {
+  serializeOut: ({ formatValue, valueMode }) => {
     return {
       tooltipSettings: {
         formatValue,
       },
+      valueMode: valueMode,
```
Comment (apryka) [inline src/charts/circular/transform.ts:20]: just `valueMode`

File: src/components/ComponentSettings/ComponentSettings.tsx (hunk 1 above)
Comment (dariuszlacheta) [reply src/components/ComponentSettings/ComponentSettings.tsx]: True... forgot to change it...

**Review by apryka — APPROVED** [review]

File: src/charts/circular/transform.ts (hunk 2 above)
Comment (dariuszlacheta) [reply src/charts/circular/transform.ts]: Changed

**Pull request merged**